coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 0.0
+    "covered_percent": 29.17
```

Comment:
This doesn't seem right, we should have either 100% or 0% since there are no tests